libraries/botframework-schema/etc/botframework-schema.api.md

```diff
@@ -953,11 +953,47 @@ export interface MeetingInfo {
     organizer: TeamsChannelAccount;
 }
 
+// @public
+export type MeetingNotification = TargetedMeetingNotification;
+
+// @public
+export interface MeetingNotificationBase<T> {
+    type: string;
+    value: T;
+}
+
+// @public
+export interface MeetingNotificationChannelData {
+    onBehalfOf?: OnBehalfOf[];
+}
+
+// @public
+export interface MeetingNotificationRecipientFailureInfo {
+    errorCode: string;
+    failureReason: string;
+    recipientMri: string;
+}
+
+// @public
+export interface MeetingNotificationResponse {
+    recipientsFailureInfo?: MeetingNotificationRecipientFailureInfo[];
+}
+
+// @public (undocumented)
+export interface MeetingStageSurface<T> {
+    content: T;
+    contentType: 'task' | string;
+    surface: 'meetingStage';
+}
+
 // @public
 export interface MeetingStartEventDetails extends MeetingEventDetails {
     startTime: Date;
 }
 
+// @public
+export type MeetingSurface = MeetingStageSurface<any>;
```

Comment:
If add `extends Surface` for` interface MeetingStageSurface<T>` (full example in the conversation above), Is there a way to show the inheritance relationship between `MeetingStageSurface `and `Surface`?

Also, does it make sense to rename `MeetingSurface `in  `export type MeetingSurface = MeetingStageSurface<any>;` to `Surface`. or "MeetingStageSurface"

Reply:
`MeetingStageSurface<T> extends Surface`, simply copies the properties/type pair in `Surface` to `MeetingStageSurface<T>`, nothing more. So then there's not much reason to have a base interface since it will have only one child interface. It seems like the dotnet sdk names it `Surface`. I will make the change from `MeetingSurface` to `Surface` in the next iteration.